crates/store/re_log_types/src/path/entity_path.rs

```diff
@@ -71,7 +71,7 @@ impl std::fmt::Debug for EntityPathHash {
 
 // ----------------------------------------------------------------------------
 
-const RESERVED_NAMESPACE_PREFIX: &str = "__";
+pub(crate) const RESERVED_NAMESPACE_PREFIX: &str = "__";
```

Comment:
nitnit: `pub` is just as good here ;-)

(If you want to see me rant in public about this go check this out 😄 
https://docs.google.com/document/d/1Z3qjy3m7eAYaTsh2n-iKxLV4Hjc6wZxgukzdQOgVH1c/edit?disco=AAABeZ2ZxHE)